paddle/fluid/pir/dialect/operator/ir/ops.yaml

```
@@ -1305,6 +1305,16 @@
     func : prune_gate_by_capacity
     data_type : gate_idx
 
+- op : pull_sparse_v2
+  args : (Tensor[] ids, Tensor[] w, int embeddingdim = 11, int tableid = 0, str accessorclass = "", str ctrlabelname = "", int paddingid = 0, bool scalesparsegrad = true, str[] inputnames = {}, bool is_distributed = true)
+  output : Tensor[](out){w.size()}
+  infer_meta :
+    func : PullSparseV2InferMeta
+  kernel :
+    func : pull_sparse_v2
+    data_type : DataType::FLOAT32
+  backward : push_sparse_v2
```

```suggestion
  backward : pull_sparse_v2_grad
```
目前看问题出在这一句上，问题在于push_sparse_v2是一个反向算子，但是定义在了前向算子的yaml。
我们的方案是利用代码生成里的invoke机制来实现，首先声明pull_sparse_v2的反向是 pull_sparse_v2_grad，然后在
paddle/fluid/pir/dialect/operator/ir/ops_backward.yaml里定义pull_sparse_v2_grad，但是要像这样
```
- backward_op : pull_sparse_v2_grad
  forward : pull_sparse_v2 (...) -> ....
  args : (...)
  output : ....
  invoke : push_sparse_v2(ids, w, out_grad_in, ...)
```
需要说明的是，这里的...表明省略，不是说真的可以用...代替，目前不支持。

---

感谢，已修改